internal/api/handlers/operations_handler.go

```diff
@@ -187,23 +187,27 @@ func (h *OperationsHandler) queryFinishedOperations(ctx context.Context, schedul
 }
 
 func extractPaginationParameters(request *api.ListOperationsRequest) (uint32, uint32, error) {
-	page := request.GetPage()
-	pageSize := request.GetPerPage()
+	page := request.Page
+	pageSize := request.PerPage
 	operationStage := request.Stage
+	defaultPageValue := uint32(1)
+	defaultPageSizeValue := uint32(15)
 
-	if page == 0 {
-		page = 1
+	if page == nil {
+		page = &defaultPageValue
+	} else if *page < 1 {
+		return 0, 0, status.Error(codes.InvalidArgument, "page must be greater than 0")
 	}
 
-	if pageSize == 0 {
-		pageSize = 15
+	if pageSize == nil {
+		pageSize = &defaultPageSizeValue
 	}
```

Comment:
I think that here we need to validate the maximum number of `pageSize` as `100`. What do you think?